src/Microsoft.Tye.Core/Serialization/YamlConfigApplicationHelpers.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Tye.ConfigModel;
+using YamlDotNet.RepresentationModel;
+
+namespace Tye.Serialization
+{
+    public static class YamlConfigApplicationHelpers
```

Comment:
Please no.

Reply:
What do you mean?

Reply:
A good rule. You can remove the word `Helpers` and it's suddenly a better name.

Reply:
`ConfigApplicationParser` would be a good name

Reply:
> What do you mean?

No helpers. Pick better names or don't make classes 😆